src/Udyam/components/Events/events.js

```diff
@@ -221,6 +221,15 @@ function Events() {
                 <>PROBLEM STATEMENT</>
               )}
             </div>
+            <div className="register-link">
+              {eventData.registerLink ? (
+                <a href="https://eesiitbhu.in/dashboard#events-register" target="_blank" rel="noreferrer">
+                  REGISTER
```

Comment:
1. Define eventData.registerLink in the data array.
2. Remove target="_blank" rel="noreferrer" from this button